### DISCUSSION TITLE
Tolerations is possible in the ephemeral agents?

### DISCUSSION THREAD

**Discussion by klinux** [terrakube-io/terrakube#1576] [General]
I've seen that node selector is possible in the ephemeral agents, tolerations was implemented too?

Thank you

**Comment by alfespa17** [comment]
You can configure the following:

https://github.com/AzBuilder/terrakube/pull/1243

https://github.com/AzBuilder/terrakube/pull/1505

Toleration are not yet implemented, you could create the issue if you would like to see that feature

It will only require some small change here 

https://github.com/AzBuilder/terrakube/blob/6c2f3c219e30890e63e92bfa29aefffbf640f6a4/api/src/main/java/org/terrakube/api/plugin/scheduler/job/tcl/executor/ephemeral/EphemeralExecutorService.java#L109

**Reply by klinux** [reply]
I will try to create a PR, thank you  :)
